docs/cli/dotnet-restore.md

```diff
@@ -0,0 +1,77 @@
+dotnet-restore
+==============
+
+## NAME
+
+`dotnet-restore` - restores the dependencies and tools of a project
+
+## SYNOPSIS
+
+dotnet-restore [--source]  
+    [--packages] [--disable-parallel]  
+    [--fallbacksource] [--configfile] [--verbosity]
+    [< root >]  
+
+## DESCRIPTION
+
+`dotnet-restore` will use NuGet to restore dependencies as well as project-specific tools that are specified in the 
+project.json file. By default, the restoration of dependencies and tools will be done in parallel.
+
+In order to restore the dependencies, NuGet needs feeds where the packages are located. Feeds are usually provided via the 
+NuGet.config configuration file; a default one is present when CLI tools are installed. You can specify more feeds by 
+creating your own NuGet.config file in the project directory. Feeds can also be specified per invocation on the command line. 
+
+For dependencies, you can specify where the restored packages will be placed during the restore operation using the 
+`--packages` argument. 
+
+For project-specific tooling, `dotnet-restore` will first restore the package in which the tool is packed, and will then
+proceed to restore the tool's dependencies as specified in its project.json. 
+
+## OPTIONS
+
+`[root]` 
+    
+ A list of projects or project folders to restore. The list can contain either a path to a `project.json` file, path to `global.json` file or  folder. The restore operation will run recursivelly for all sub-directories and restore for each given project.json file it finds.
+
+`-s`, `--source` [SOURCE]
+
+Specify a source to use during the restore operation. This will override all of the sources specified in the NuGet.config file(s). 
```

Comment:
is this a multi-instance parameter? Is there a way to just add an additional source without writing my own nuget.config?